delta/app/src/main/scala/ch/epfl/bluebrain/nexus/delta/routes/DeltaDirectives.scala

```diff
@@ -290,6 +290,16 @@ trait DeltaDirectives extends RdfMarshalling with QueryParamsUnmarshalling {
   )(implicit s: Scheduler, cr: RemoteContextResolution, ordering: JsonKeyOrdering): Route =
     completeUIOOpt(io.map(Some(_)))
 
+  /**
+    * Completes a passed [[A]] with the desired output format using the implicitly available [[JsonLdEncoder]].
+    *
+    * @param value  the value to be returned
+    */
+  def completePure[A: JsonLdEncoder](
```

Comment:
You don't want to call it just `complete` to avoid interfering with the Akka http one?

Reply:
Exactly.